src/main/java/org/team1540/robot2025/subsystems/intake/CoralIntakeIOReal.java

```diff
@@ -18,37 +22,38 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.units.measure.*;
 
-public class IntakeIOReal implements IntakeIO {
-
+public class CoralIntakeIOReal implements CoralIntakeIO {
```

Comment:
I think it really makes sense, for standardization, to just call them all `Real` rather than `TalonFX`. Unless you want to call this `CoralIntakeIOTalonFXPivotSparkMaxFunnelTalonFXRoller` or split it into three IO classes

Reply:
The problem with naming everything "real" is that now it's unclear what hardware the IO is designed for, and you have to now make things inconsistent for IOs that have several option (like the grabber sensor IOs).
It is annoying that this one is inconsistent since it has a mix of hardware, but like I don't think it matters very much. You're going to have an inconsistency somewhere, it's either here or with the grabber sensor IOS.

Reply:
your PR says "consistency changes" 🤷

It doesn't matter what hardware the IO is designed for unless you have multiple options (Only the grabber sensor IOs and DT gyro IOs). None of the motor IOs have interchangeable types

Reply:
Seems like quite a big change. I'm probably not going to do that for this PR, but if it matters enough to you, you can make another PR for this.